### DISCUSSION TITLE
upgrade kernel to linux-6.1.7. (kernel.org) from Linux 5.15.83.1-microsoft-standard-WSL2

### DISCUSSION THREAD

**Discussion by valorisa** [microsoft/WSL#9517] [General]
Hello,
Possible or not ? I read some informations this previous days and if Linux 5.15.83.1-microsoft-standard-WSL2 is eligible for this operation. If so, how do we proceed ?

https://cdn.kernel.org/pub/linux/kernel/v6.x/linux-6.1.7.tar.xz

Thanks.

**Comment by Biswa96** [comment]
This discussion may help https://github.com/microsoft/WSL/discussions/9496
